recipe/patch_yaml/openff-bespokefit-patch.yaml

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
```suggestion
```

Reply:
This extra line is triggering the linter.

Reply:
Thanks!